frontend/src/modules/settings/containers/DisplayTab.tsx

```diff
@@ -189,6 +243,12 @@ export const DisplayTab = ({ onSettingChange }: Props): JSX.Element => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [date, period, minute, hour, day, month, year, unit, theme, brightness]);
 
+  /**
+   * function for handling month change.
```

Comment:
What is this function used for? i.e. what does this do when it's called to handle the  month change?

Reply:
This function is callback from `ValueController`. Whenever user changes value of `Month` setting, it updates the local state in the component in `handleMonthChange`